storage/operation/prefix.go

```diff
@@ -0,0 +1,150 @@
+//nolint:golint,unused
+package operation
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const (
```

Comment:
where did these constants come from? Is this a unrelated thing?

Reply:
They are copied from https://github.com/onflow/flow-go/blob/master/storage/badger/operation/prefix.go